guides/common/attributes-satellite.adoc

```diff
@@ -11,6 +11,7 @@
 :AdministeringDocURL: {BaseURL}administering_red_hat_satellite/index#
 :ConfiguringLoadBalancerDocURL: {BaseURL}configuring_capsules_with_a_load_balancer/index#
 :ContentManagementDocURL: {BaseURL}managing_content/index#
+:ConversionsApplianceDocURL: {BaseURL}converting_hosts_to_rhel_by_using_conversions_appliance/index#
```

Comment:
I wonder if this should be moved to `attributes-base` if for any reason someone wants to link it outside the satellite build. On the other hand it is better categorized here. What do you think?

Reply:
That's a good point. Would it be worth moving all guides that aren't upstreamized yet to `attributes-base`?
@adamlazik1 @maximiliankolb Opinions?

Reply:
For me it seems like a good change but it is also a non-blocking suggestion.

Reply:
Yeah, works for me.

Reply:
Unfortunately, there's no easy way of including RH links in `attributes-base`, because RH base URL includes Satellite version number and attributes-base doesn't define Satellite version. So I'm leaving it as it is for now.